app/components/post_draft/send_action/scheudled_post_tooltip.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Image} from 'expo-image';
+import React from 'react';
+import {StyleSheet, TouchableOpacity, View} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import FormattedText from '@components/formatted_text';
+import {Preferences} from '@constants';
+import {changeOpacity} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    onClose: () => void;
+}
+
+const longPressGestureHandLogo = require('@assets/images/emojis/swipe.png');
+
+const hitSlop = {top: 10, bottom: 10, left: 10, right: 10};
+
+const styles = StyleSheet.create({
+    container: {
+        paddingHorizontal: 4,
+        paddingVertical: 8,
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 18,
+    },
+    close: {
+        position: 'absolute',
+        top: 0,
+        right: 0,
+    },
+    description: {
+        color: Preferences.THEMES.denim.centerChannelColor,
```

Comment:
Why is this using denim explicitly and not based on the selected theme ?

Reply:
The tooltip component we use, unfortunately, doesn't support changing the background color of the tooltip. Its no longer on development and has same open issue. So, similar to what we have in drafts, we just use Denim theme colors because they work well with the forced white background color of the tooltip.

![Screenshot 2025-02-17 at 11 49 54 AM](https://github.com/user-attachments/assets/4b1757c3-e774-43ad-9e0e-558dd3cfcec0)